src/PlatformServices/Services/TestDataSource.cs

```diff
@@ -47,11 +55,25 @@ public IEnumerable<object> GetData(UTF.ITestMethod testMethodInfo, ITestContext
 
             try
             {
+
+/* Unmerged change from project 'PlatformServices (net462)'
```

Comment:
There was an error applying some codefix - you want to review and fix this line.

Reply:
This was here already, need to make sure before removal.